tests/CreatePaymentTest.php

```diff
@@ -123,6 +106,6 @@ public function testCreateApiPayment()
 
         $result = $this->client->createPayment($createPayment);
 
-        static::assertTrue(get_class($result) === 'ThePay\ApiClient\Model\CreatePaymentResponse');
+        self::assertTrue(get_class($result) === 'ThePay\ApiClient\Model\CreatePaymentResponse');
```

Comment:
Tady by se možná hodilo nahradit ten řetězec za `CreatePaymentResponse::class`

Reply:
Navíc místo assertTrue by se zde spíše hodilo použít např. `self::assertSame($result::class, CreatePaymentResponse::class);`, aby to v případě failu vrátilo relevantnější informace.